srcpkgs/python-pywatchman/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-pywatchman'
+pkgname=python-pywatchman
```

Comment:
The py prefix isn't required if you start with `python-`.

Some examples (us -> upstream):

- python-openssl -> pyOpenSSL.
- python3-xdg -> pyxdg
- python3-cups -> pycups
- python3-curl -> pycurl